CSS/CSS.sublime-syntax

```diff
@@ -1502,6 +1198,7 @@ contexts:
   other-property:
     # Note: Consume unknown identifiers to maintain word boundaries.
```

Comment:
Comment is not accurate anymore since we now actively consume remaining properties to assign a scope.